src/Raven.Server/Documents/PeriodicBackup/BackupUploader.cs

```diff
@@ -259,6 +259,7 @@ private string CombinePathAndKey(string path)
 
             var localUploadStatus = uploadStatus;
             var threadName = $"Upload backup file of database '{_settings.DatabaseName}' to {targetName} (task: '{_settings.TaskName}')";
+            var shortThreadName = $"UBF {_settings.DatabaseName} t {targetName}";
```

Comment:
No way that I'll remember what that means in the future.
In general, we need short, but readable.

Linux thread names are 14 chars, can use a prefix that is unique so we can identify, but generally still must be readable